### ISSUE TITLE
Cannot unmarshal widget of type: uptime

### ISSUE THREAD

**Issue by marekaf** [terraform-providers/terraform-provider-datadog#327]
### Terraform Version
```
Terraform v0.12.9
+ provider.aws v2.28.1
+ provider.datadog v2.4.0
```

### Affected Resource(s)
- datadog_dashboard

### Terraform Configuration Files
```hcl
resource "datadog_dashboard" "coredns" {
  is_read_only = false
  layout_type  = "free"
  notify_list  = []
  title        = "CoreDNS dashboard"

  widget {

  }
}
```

### Expected Behavior
Successful import of datadog_dashboard resource

### Actual Behavior
```
terraform import datadog_dashboard.coredns <ID_OF_MY_DASHBOARD>
Acquiring state lock. This may take a few moments...
datadog_dashboard.coredns: Importing from ID "<ID_OF_MY_DASHBOARD>"...

Error: Cannot unmarshal widget of type: uptime


Releasing state lock. This may take a few moments...
```

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform import datadog_dashboard.coredns <ID_OF_MY_DASHBOARD>`

### Important Factoids
I think the problem might be that this dashboard contains the SLO (beta) widget

**Comment by Kydybets** [comment]
Got the same problem

**Comment by zioproto** [comment]
I was able to workaround the problem in this way:
* Delete all dashboards manually from the Datadog website
* Delete all `datadog_dashboard` resources from the terraform code.
* Run Terraform
* Reintroduce all the `datadog_dashboard` resources in the terraform code
* Run Terraform again

**Comment by Kydybets** [comment]
so in general if you have some dashboard, you need manually move it into terraform code

**Comment by TheKevJames** [comment]
I think the problem here is that the `uptime` widget (ie. the beta SLO thing) is not yet supported in the terraform bindings (see related issue #179).

**Comment by bkabrda** [comment]
Hey folks, this should be fixed in release 2.6.0 of the provider. Could you please give it a go and let me know if it works fine for you? Thanks!

**Comment by gzussa** [comment]
This issue should be fixed by now, please try to use the `slo` widget
